Makefile

```diff
@@ -18,7 +18,7 @@ DOCKER_IMAGE_CI ?= quay.io/gravitational/teleport-ci
 
 # These are standard autotools variables, don't change them please
 ifneq ("$(wildcard /bin/bash)","")
```

Comment:
I'm not sure I follow this line, isn't `$(wildcard /bin/bash)` always `/bin/bash`?

Same throughout.

Reply:
`wildcard` doesn't fallback to using the pattern the way normal expansion does.  It emits an empty string if there are no matches.  This lets it function as an existence check.